src/framework/standard/parse/mod.rs

```diff
@@ -371,6 +374,7 @@ pub fn command(
                     check_discrepancy(ctx, msg, config, &group.options)?;
 
                     return res;
+                    is_prefixless = true;
```

Comment:
The assignment ought to appear before the return statement. It's unreachable code otherwise.
```suggestion
                    is_prefixless = true;
                    return res;
```